### DISCUSSION TITLE
对于样式的建议

### DISCUSSION THREAD

**Discussion by NoverNobida** [usememos/memos#43] [General]
1. 在未输入以及没有超出输入框的情况下, 输入框不应该有滚动条. 因为Memos的特点应该是短文本输入比较多, 有滚动条影响感观.
![图片](https://user-images.githubusercontent.com/46912250/161366383-fc903d1f-8f47-4178-a8b8-485c91be36bf.png)
2. 整个页面分为三块实际上极大地浪费了空间, 显得松散而不紧凑, 特别是第二块仅仅只是一个输入框, 存在较多空区域. 在对历史memo进行编辑时, 选择要编辑的memo, 却要跳到左边编辑(而不是右边), 目光要在第二块第三块之间来回调整, 既不符合从左到右的编辑习惯, 同时割裂感和不集中感严重. 这种调整是不是出于"尽量和flomo设计得不一样"考虑的? 但是实际上原先的设计, 也就是仿flomo的设计比较科学, 整体就是两列, 左边一些基础信息, 以及tag等, 右边就是最上面输入框, 输入框下面就是一条一条的笔记. 这样既紧凑又高效.
3. 希望对mobile端的样式加以适配. 实际上手机端要适配, 输入框下面还是笔记列, 和之前的样式实际上是一样的, 所以不太明白对于样式的大调整是出于什么考虑.

**Comment by rebron1900** [comment]
奇怪，我这边没看到过滚动条，你什么浏览器和系统？

**Comment by NoverNobida** [comment]
Windows 11, Chrome, edge和Firefox都有. 
![图片](https://user-images.githubusercontent.com/46912250/161413101-e9ed94e2-e300-4486-9eb1-6b9cdbd6640e.png)
![图片](https://user-images.githubusercontent.com/46912250/161413128-1e03eb37-7070-4e73-b203-0f830dd8f432.png)
![图片](https://user-images.githubusercontent.com/46912250/161413150-23250bb7-43f8-48e8-9307-7a4b71d9617a.png)

在Firefox加了
```css
.common-editor-inputer {
  scrollbar-width: none;
}
```
之后
![图片](https://user-images.githubusercontent.com/46912250/161413255-4a112400-7710-474d-a481-4a5c9b720853.png)
说明是本身样式问题,而不是浏览器或系统原因

**Comment by boojack** [comment]
尝试利用全部空间进行内容布局，而不是像 flomo 那样左右会留出空白。参考：锤子笔记、语雀小记

**Reply by rebron1900** [reply]
我docker上用了自动更新，不会到时候自动更新把数据都丢失了把。。

**Comment by rebron1900** [comment]
哦，我部署的还是老版本。
